components/EditOnGithub.js

```diff
@@ -0,0 +1,31 @@
+import { useRouter } from "next/router";
+import { OutlineButton } from "./Button";
+
+export default function EditOnGitHub() {
+  const router = useRouter();
+  const { asPath } = router;
+
+  const githubUrl = `https://github.com/EddieHubCommunity/LinkFree/edit/main/pages${
+    asPath === "/docs" ? `/docs/index.js` : `${asPath}.mdx`
+  }`;
+
+  return (
+    <div className="mt-12 border border-primary-low-medium rounded-lg p-4">
+      <a href={githubUrl} target="_blank" rel="noopener noreferrer">
```

Comment:
Can our `Link` component be used here?

Reply:
I tried Link component for the same but dropped the idea due to less visibility for the text in the dark mode. I was about to propose this problem.